src/main/java/picard/sam/markduplicates/MarkDuplicatesForFlowHelper.java

```diff
@@ -236,8 +273,8 @@ static protected int getFlowSumOfBaseQualities(final SAMRecord rec, final int th
         final byte[]  bases = rec.getReadBases();
 
         // create iteration range and direction
-        final int startingOffset = !rec.getReadNegativeStrandFlag() ? 0 : bases.length;
-        final int endOffset = !rec.getReadNegativeStrandFlag() ? bases.length : 0;
+        final int startingOffset = !rec.getReadNegativeStrandFlag() ? 0 : bases.length - 1;
```

Comment:
was this previously a bug? I don't see how this change is related to allowing a range in the start position or changing the quality score calculation.

Reply:
Was a bug indeed